webapp/src/components/views/BackendProbe.tsx

```diff
@@ -44,14 +44,17 @@ export const BackendProbe: FC<IData> = ({ uri, onBackendFound }) => {
                     return;
                 }
 
-                const json: unknown = await result.json();
-
-                if (json === null) {
-                    dispatch(setMaintenance(false));
-                    onBackendFound();
-                }
-
-                model.current = json as IMaintenance | null;
+                result
+                    .json()
+                    .then((data) => {
+                        // Has payload is maintenance
```

Comment:
nit: comment is hard to undertand gramatically

Reply:
Updated